hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterHoodieTableRenameCommand.scala

```diff
@@ -46,6 +45,15 @@ class AlterHoodieTableRenameCommand(
 
       // Call AlterTableRenameCommand#run to rename table in meta.
       super.run(sparkSession)
+
+      // update table properties path in every op
```

Comment:
Shall we check and apply below logic only when it's a 'MANAGED' table -- thus to save some remote calls to HMS ?

Reply:
@jinxing64 yes, if the table type is external, the path I think will not be update, but if the table type is managed, the path has two situation, update or not depend on if the table set location or not when we create table, and I found it's diff in diff spark vertion, that's why mine ut case judge spark version.

so, I think table type is not the better condition to judge update tblp 'path', maybe has other. For correctness, I overwrite path by sds path in every op